contracts/0.8.25/vaults/VaultFactory.sol

```diff
@@ -0,0 +1,109 @@
+// SPDX-FileCopyrightText: 2024 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+import {UpgradeableBeacon} from "@openzeppelin/contracts-v5.0.2/proxy/beacon/UpgradeableBeacon.sol";
+import {BeaconProxy} from "@openzeppelin/contracts-v5.0.2/proxy/beacon/BeaconProxy.sol";
+import {Clones} from "@openzeppelin/contracts-v5.0.2/proxy/Clones.sol";
+
+import {IStakingVault} from "./interfaces/IStakingVault.sol";
+
+pragma solidity 0.8.25;
+
+interface IDelegation {
```

Comment:
why we don't import this interface though?

Reply:
let's add the comment that it's intended (to hook up the particular interface strictly)